src/pages/patientView/PatientViewPage.tsx

```diff
@@ -43,6 +45,7 @@ interface IPatientViewState {
     mutationData: any;
     mrnaExprRankData?: MrnaRankData;
     activeTabKey: number;
+    hotspotsData: any;
```

Comment:
how do you feel about the convention I started w MrnaRankData - define the type?